Source/WebCore/crypto/mac/CryptoKeyRSAMac.cpp

```diff
@@ -131,9 +131,14 @@ RefPtr<CryptoKeyRSA> CryptoKeyRSA::create(CryptoAlgorithmIdentifier identifier,
         return nullptr;
 
     CCRSACryptorRef cryptor = nullptr;
-    // FIXME: It is so weired that we recaculate the private exponent from first prime factor and second prime factor,
+    // FIXME(nitin: Does not need to be now): It is so weired that we recaculate the private exponent from first prime factor and second prime factor,
     // given the fact that we have already had it. Also, the re-caculated private exponent may not match the given one.
     // See <rdar://problem/15452324>.
+    //    I spent some time to fix this and the CCRSACryptorRecoverPrivateKey in CommonCrypto actually works with n,e,d but
+    //    I also saw that chrome and firefox don't support just d without additional params.
```

Comment:
We also typically don't use first person, it being unclear who "I" is referring to.  

CCRSACryptorRecoverPrivateKey in CommonCrypto actually works with n,e,d but
Chrome and Firefox don't support just d without additional params.

Reply:
comment updated.